exercise_04.py

```diff
@@ -0,0 +1,23 @@
+def found_pi(number):
+    formula = 0
+
+    for i in range(1, number + 1):
+        formula += (-1) ** ( i + 1) / (2 * i - 1)
+        pi = 4 * formula
+
+    return f"{pi:.4f}"
+
+
+def pi_values_table(n):
```

Comment:
@DanielVelinrb , a function should have a verb that indicates what it does. What if we use `print_pi_values_table` since it is more accurate?